tests/run_integration.sh

```diff
@@ -19,6 +19,17 @@ cleanup () {
     docker stop $MONGO_CONTAINER && docker rm $MONGO_CONTAINER
 }
 
+# Docker on non-linux runs in a vm. This attempts to detect that,
+# ensure there is a known vm docker can use. Then uses it.
+# This kinda sucks, but works on Mac OS at least.
+if [[ $(docker-machine -v) ]]; then
```

Comment:
Let's change this line to:
`if docker-machine -v 2>/dev/null; then`
The brackets aren't needed for a command test and we should suppress error messages in this case.